src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -354,6 +354,107 @@ featurizers:
           - type: uint8
             name: isPaidTimeOff
 
+  # ----------------------------------------------------------------------
+  - name: ForecastingPivotFeaturizer
+    estimator_name: GrainedForecastingPivotEstimator

```

Comment:
I dont think this featurizer needs to be grain aware. It has no state.
It also needs to have dynamic output.

Reply:
I agree - pending checkin.